.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.8 (condaforge)" project-jdk-type="Python SDK" />
```

Comment:
Line 3 specifies the interpreter name. Since it seems everyone has different names for their interpreter we should add this to the git ignore. See my comment on .gitignore.

Reply:
Actually, I think everyone will just have to change their interpreter name to "Python 3.8 (condaforge)". Adding misc.xml to the git ignore won't change what interpreter pycharm tries to use for the project.